templates/navigation/partials/_preview-links.html

```diff
@@ -5,9 +5,9 @@
   <div class="row u-no-padding">
     {% for link in preview_secondary_links.links %}
       <div class="col-2">
-        <a class="p-link--inverted" href="">
+        <a class="p-link--inverted" href="https://ubuntu.com/engage/sicredi-openstack-case-study">
           <div class="p-navigation--image-wrapper">
-            <img src="{{ link.image_url }}" alt="{{ link.alt_text }}" />
+            <!-- <img src="{% if link.image_url == '/PLACE-HOLDER' %}https://variety.com/wp-content/uploads/2021/07/Rick-Astley-Never-Gonna-Give-You-Up.png?w=1000&h=667&crop=1{% endif %}" alt="" /> -->
```

Comment:
I think the link should be [this](https://www.youtube.com/watch?v=dQw4w9WgXcQ) instead

Reply:
firstly, omg.. it's been a long time since I fell for that one.
secondly, oops forgot to remove my test images hahahaha